[DOCS] Reformats settings in drilldown doc (#120299)
elastic/kibana: docs/settings/url-drilldown-settings.asciidoc

Configure the URL drilldown settings in your `kibana.yml` configuration file.

[[external-URL-policy]] `externalUrl.policy`::
Configures the external URL policies. URL drilldowns respect the global *External URL* service, which you can use to deny or allow external URLs.
By default all external URLs are allowed.

> 

Author: ```suggestion
+
```

For example, to allow only external URLs to the `example.com` domain with the `https` scheme, except for the `danger.example.com` sub-domain,
which is denied even when `https` scheme is used:

> 

Author: ```suggestion
+
```

Configure the URL drilldown settings in your `kibana.yml` configuration file.

[[external-URL-policy]] `externalUrl.policy`::
Configures the external URL policies. URL drilldowns respect the global *External URL* service, which you can use to deny or allow external URLs.
By default all external URLs are allowed.

> This example allows external URLs only to the `example.com` domain with the `https` scheme, except for the `danger.example.com` sub-domain,

Author: ```suggestion
For example, to allow only external URLs to the `example.com` domain with the `https` scheme, except for the `danger.example.com` sub-domain,
```

> which is denied even when `https` scheme is used.

Author: ```suggestion
which is denied even when `https` scheme is used:
```
